Automattic/sensei
# Organize course outline parent block

Contains the organizational bits from #3635, without the unsupported block.json metadata loading.
Also fixes an issue with a placeholder icon.

### Changes proposed in this Pull Request

* Move the Course outline parent block files into a folder
* Move block registration out from the block itself — this could help with testing

### Testing instructions

* Make sure Outline block in editor still works
* Create a new course, make sure the placeholder looks right


### Screenshot / Video

This is fixed now, but was beautiful:

<img width="647" alt="image" src="https://user-images.githubusercontent.com/176949/93827024-c0394b00-fc68-11ea-8332-cb017ab313df.png">
